ckanext/security/resource_upload_validator.py

```diff
@@ -82,13 +82,13 @@ def validate_upload_type(resource):
     _add_mimetypes()
     extensions_and_mimetypes = _build_mimetypes_and_extensions(filename, uploaded_file)
 
-    config_blacklist = eval(config.get('ckanext.security.upload_blacklist', '[]'))
+    config_blacklist = eval(config.get('ckanext.security.upload_blacklist', '[]').lower())
```

Comment:
config blacklist value is a string initially so we can lowercase the whole thing then eval into an array.